src/include/sof/drivers/ipc.h

```diff
@@ -143,7 +143,7 @@ int ipc_stream_send_xrun(struct comp_dev *cdev,
 	struct sof_ipc_stream_posn *posn);
 
 int ipc_queue_host_message(struct ipc *ipc, uint32_t header, void *tx_data,
-			   size_t tx_bytes, uint32_t replace);
+			   size_t tx_bytes, bool replace);
```

Comment:
Change from uint32_t to bool is a refactor, I'd prefer it in a separate commit dedicated to all such refactors.